README.md

```diff
@@ -96,4 +96,30 @@ make
 hack/run-external.sh <kubectl cluster name>
 ```
 
+## Removal
+
+To remove the operator and prometheus, first delete any third party resources you created. The
+operator will automatically shut down and remove prometheus and alertmanager pods, and associated configmaps.
+
+```
+kubectl delete prometheus,servicemonitor,alertmanager --all
+```
+
+After a couple of minutes you can go ahead and remove the operator itself.
+
+```
+kubectl delete -f deployment.yaml
+```
+
+The operator automatically creates two services and defines three third party resources. You can clean these up now.
+
+```
+kubectl delete --ignore-not-found service prometheus-operated alertmanager-operated
```

Comment:
these are created in each namespace and should then also be removed from all namespaces